lib/aposResponse.js

```diff
@@ -7,10 +7,12 @@ export default async function aposResponse(req) {
   const aposUrl = new URL(url.pathname, aposHost);
   aposUrl.search = url.search;
 
+
+  const excludedHeadersLower = new Set(config.excludeRequestHeaders?.map(h => h.toLowerCase()) || []);
```

Comment:
Why not move this to top level code in this file so it doesn't have to happen on every request?

Reply:
Done.